src/components/card/CommentItem.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+	<li>
+		<hr>
+		<form v-if="edit" @submit.prevent="updateComment">
+			<input v-model="commentMsg" type="text" autofocus
```

Comment:
This should also use the rich editor with mentions. Please move that to a separate component so it can be reused.